cluster/gce/config-default.sh

```diff
@@ -544,14 +562,17 @@ PROXY_NAME="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
 PROXY_TAG="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
 ARKTOS_SCALEOUT_PROXY_APP="${ARKTOS_SCALEOUT_PROXY_APP:-haproxy}"
 
-# true if this is a resource partition cluster
-KUBERNETES_RESOURCE_PARTITION="${KUBERNETES_RESOURCE_PARTITION:-false}"
-# true if this is a tenant partition cluster
-KUBERNETES_TENANT_PARTITION="${KUBERNETES_TENANT_PARTITION:-false}"
+SCALEOUT_CLUSTER=${SCALEOUT_CLUSTER:-false}
+SCALEOUT_TP_COUNT="${SCALEOUT_TP_COUNT:-1}"
+SCALEOUT_RP_COUNT="${SCALEOUT_RP_COUNT:-1}"
+HAPROXY_TLS_MODE=${HAPROXY_TLS_MODE:-"bridging"}
```

Comment:
can we separate the haproxy related code to a different file, i.e. to set up haproxy on top of the kube-up logic ? this will make the deployment setup much easy to understand and more manageable

Reply:
Agree with you. For now, we don't have a dedicated files for proxy env, but all proxy env is  under its own setup function. Will move this to correct position.

Reply:
move this to proxy setup function.